src/styles/window-top.sass

```diff
@@ -34,6 +44,11 @@
 
       &.maximize
         background: #62C655
+        padding-top: 2px
+
+  .window-top-container__left:hover strong
```

Comment:
can you change this for separate hover, not that all icons change when hovering left corner?